internal/stores/accounts.go

```diff
@@ -29,10 +30,41 @@ func (dbAccount) TableName() string {
 	return "ephemeral_accounts"
 }
 
+// Accounts returns all accoutns from the db.
 func (s *SQLStore) Accounts() ([]api.Account, error) {
-	return []api.Account{}, nil // TODO: implement
+	var dbAccounts []dbAccount
+	if err := s.db.Find(&dbAccounts).Error; err != nil {
+		return nil, err
+	}
+	accounts := make([]api.Account, len(dbAccounts))
+	for i, acc := range dbAccounts {
```

Comment:
`nit` can you add a `convert` on `dbAccount` for consistency with the other types?